README.md

```diff
@@ -6,32 +6,40 @@ This controller operates self-hosted runners for GitHub Actions on your Kubernet
-  - [Runner Deployments](#runnerdeployments)
-    - [Note on scaling to/from 0](#note-on-scaling-tofrom-0)
-  - [Autoscaling](#autoscaling)
-    - [Webhook Driven Scaling](#webhook-driven-scaling)
-    - [Autoscaling to/from 0](#autoscaling-tofrom-0)
-    - [Scheduled Overrides](#scheduled-overrides)
-  - [Runner with DinD](#runner-with-dind)
-  - [Additional Tweaks](#additional-tweaks)
-  - [Runner Labels](#runner-labels)
-  - [Runner Groups](#runner-groups)
-  - [Using IRSA (IAM Roles for Service Accounts) in EKS](#using-irsa-iam-roles-for-service-accounts-in-eks)
-  - [Stateful Runners](#stateful-runners)
-  - [Ephemeral Runners](#ephemeral-runners)
-  - [Software Installed in the Runner Image](#software-installed-in-the-runner-image)
-  - [Common Errors](#common-errors)
+- [actions-runner-controller](#actions-runner-controller)
+  - [Motivation](#motivation)
+  - [Installation](#installation)
+    - [GitHub Enterprise Support](#github-enterprise-support)
+  - [Setting Up Authentication with GitHub API](#setting-up-authentication-with-github-api)
+    - [Deploying Using GitHub App Authentication](#deploying-using-github-app-authentication)
+    - [Deploying Using PAT Authentication](#deploying-using-pat-authentication)
+    - [Deploying Multiple Controllers](#deploying-multiple-controllers)
+  - [Usage](#usage)
+    - [Repository Runners](#repository-runners)
+    - [Organization Runners](#organization-runners)
+    - [Enterprise Runners](#enterprise-runners)
+    - [RunnerDeployments](#runnerdeployments)
+        - [Note on scaling to/from 0](#note-on-scaling-tofrom-0)
+    - [Autoscaling](#autoscaling)
+      - [Webhook Driven Scaling](#webhook-driven-scaling)
+        - [Example 1: Scale on each `workflow_job` event](#example-1-scale-on-each-workflow_job-event)
+        - [Example 2: Scale up on each `check_run` event](#example-2-scale-up-on-each-check_run-event)
+        - [Example 3: Scale on each `pull_request` event against a given set of branches](#example-3-scale-on-each-pull_request-event-against-a-given-set-of-branches)
+      - [Autoscaling to/from 0](#autoscaling-tofrom-0)
+      - [Scheduled Overrides](#scheduled-overrides)
+    - [Runner with DinD](#runner-with-dind)
+    - [Additional Tweaks](#additional-tweaks)
+    - [Runner Labels](#runner-labels)
+    - [Runner Groups](#runner-groups)
+    - [Using IRSA (IAM Roles for Service Accounts) in EKS](#using-irsa-iam-roles-for-service-accounts-in-eks)
+    - [Use with Istio](#use-with-istio)
+    - [Stateful Runners](#stateful-runners)
+    - [Ephemeral Runners](#ephemeral-runners)
+    - [Software Installed in the Runner Image](#software-installed-in-the-runner-image)
+    - [Common Errors](#common-errors)
+      - [invalid header field value](#invalid-header-field-value)
+      - [Runner coming up before network available](#runner-coming-up-before-network-available)
+      - [Runner coming up before dind container is ready](#runner-coming-up-before-dind-container-is-ready)
```

Comment:
Remove this structure change please, headings after a certain point become too small and so we don't want a root heading they all sit under

Reply:
Will do. I think my formatter did it by default.